roles/redpanda_broker/tasks/install-rp-deb-airgap.yml

```diff
@@ -0,0 +1,22 @@
+- name: Copy Redpanda DEBs tarball to host
+  ansible.builtin.copy:
+    src: "{{ airgap_copy_src }}/redpanda_debs.tar.gz"
+    dest: "{{ airgap_copy_dest }}/redpanda_debs.tar.gz"
+
+- name: Unpack Redpanda DEBs on host
+  ansible.builtin.unarchive:
+    src: "{{ airgap_copy_dest }}/redpanda_debs.tar.gz"
+    dest: "{{ airgap_copy_dest }}"
+
+- name: Install Redpanda DEBs
+  ansible.builtin.shell: "dpkg -i --force-confold {{ airgap_copy_dest }}/*.deb"
+  become: true
+
+- name: Ensure /var/lib/redpanda and all contents are owned by redpanda:redpanda
```

Comment:
we should enforce a permission here. at very least, 0770.

Reply:
I think /var/lib/redpanda should be empty at this point, not sure we need to recurse to set everything. this could cause long chown runs once the directory is fully populated.

Reply:
Ansible defaults to enforcing the system default permissions which IMO is preferable here since we don't have a specific need.